src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,10 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
+    ))}
```

Comment:
When rendering a list of components using `map`, each child in the list should have a unique `key` prop. Consider adding a `key` prop to the `CommentInfo` component, using a unique identifier from the `comment` object, such as `comment.id`.